user-guide/Troubleshooting/Procedures/Investigating_OpenSearch_Issues.md

```diff
@@ -105,6 +105,12 @@ To fix this issue:
 > [!IMPORTANT]
 > Deleting the nodes folder will result in loss of data. You should only do so with a new installation of OpenSearch.
 
+## OpenSearch logging mentions unknown setting [node.voting only]
+
+When [node.voting only] was configured for your tie-breaker in *opensearch.yml*, this will not work because OpenSearch has this functionality, but appears to have blocked the implementation when it was forked from Elasticsearch.
```

Comment:
This sentence is not clear. Do you mean that node.voting only is an Elasticsearch functionality that will not work in OpenSearch?

Reply:
Reworked it a bit.

Indeed it will not work as we found out with a customer project.